test/Libraries/Microsoft.Extensions.Telemetry.Testing.Tests/Metering/MetricCollectorTests.cs

```diff
@@ -366,6 +366,10 @@ public static async Task Dispose()
         await Assert.ThrowsAsync<ObjectDisposedException>(async () => await collector.WaitForMeasurementsAsync(1));
         await Assert.ThrowsAsync<ObjectDisposedException>(async () => await collector.WaitForMeasurementsAsync(1, TimeSpan.FromSeconds(1)));
 
+        // HACK: there seems to be something executing asynchronously which takes a while to finish. This needs to be tracked on
+        //       I'm adding the sleep here to keep the test from being flaky.
```

Comment:
Do we need a tracking issue for that?

Reply:
Yes, I am on it. I you want to log issue for that will not hurt too.

Reply:
@geeknoid has the fix in the PR https://github.com/dotnet/extensions/pull/4121